client/src/App/App.test.tsx

```diff
@@ -11,7 +11,7 @@ describe('App', () => {
   it('renders a heading', async () => {
     render(<App />);
     await waitFor(
-      () => expect(screen.getByRole('heading')).toHaveTextContent('trshcmpctr')
+      () => expect(screen.getByRole('heading', { level: 1 })).toHaveTextContent('trshcmpctr')
```

Comment:
I can't figure out where the came from; either major upgrade of eslint-plugin-react-hooks or react-router-dom?